src/lib/responses/error.response.ts

```diff
@@ -0,0 +1,22 @@
+import { Response } from "express";
+
+interface ErrorInterface {
+    error: string,
+    statusCode: number,
+    timestamp: number
+}
+
+function errorResponse(res: Response, statusCode: number, message: string): void {
+    const errorMessage = {
```

Comment:
this error function does 2 things:
1.  create the error message
2. send it via the response message

I think we should decouple those. the middleware knows about express and should be doing the `res.send`, and these error utils should only deal with creating errors.